torch/csrc/api/include/torch/jit.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include <string>
+#include <torch/csrc/jit/script/module.h>
+#include <torch/csrc/jit/script/compiler.h>
+#include <torch/csrc/jit/stack.h>
+
+namespace torch {
+namespace jit {
+
+/// Compiles Python JIT code into a graph that can be executed.
+///
+/// For example:
+/// ```
+/// auto module = torch::jit::compile(R"JIT(
+///   def relu_script(a, b):
+///     return torch.relu(a + b)
+///   def test_while(a, i):
+///     while i < 10:
+///       a += a
+///       i += 1
+///     return a
+/// )JIT");
+/// auto output = torch::jit::run(module, "relu_script", a, b);
+/// auto output = torch::jit::run(module, "test_while", a, b);
+/// ```
+///
+/// @param source A JIT string containing functions that are valid under the
```

Comment:
I think a "JIT string" isn't really a thing, it's just a string. Also "under the scope of the script compiler" isn't too clear to me. Maybe just "A string containing the script code to compile".